src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="home-page__content">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="IMac"
+        class="card__img"
+      >
+      <div class="card__name">
```

Comment:
due to checklist we have to use semantics (h2, p) with classes for text
here and further

Reply:
Yeah, but they have their own default values for padding and margin, so I've just decided to use span instead